src/commands/classified/internal.rs

```diff
@@ -8,16 +8,20 @@ use nu_protocol::{CommandAction, Primitive, ReturnSuccess, UntaggedValue, Value}
 pub(crate) async fn run_internal_command(
     command: InternalCommand,
     context: &mut Context,
-    input: InputStream,
+    input: Option<InputStream>,
     source: Text,
-) -> Result<InputStream, ShellError> {
+) -> Result<Option<InputStream>, ShellError> {
     if log_enabled!(log::Level::Trace) {
         trace!(target: "nu::run::internal", "->");
         trace!(target: "nu::run::internal", "{}", command.name);
         trace!(target: "nu::run::internal", "{}", command.args.debug(&source));
     }
 
-    let objects: InputStream = trace_stream!(target: "nu::trace_stream::internal", "input" = input);
+    let objects: InputStream = if let Some(input) = input {
+        trace_stream!(target: "nu::trace_stream::internal", "input" = input)
+    } else {
+        InputStream::empty()
```

Comment:
Is this a good default?

Reply:
It's probably fine to match what we had previously, which I think this does?

Reply:
Yep, this does match what we had before 👍